components/upload/Upload.razor

```diff
@@ -57,89 +57,90 @@
                 }
 
                 <Tooltip Title="tips" Disabled="file.State != UploadState.Fail" Class="@containerClass" Style="@(IsPictureCard ? "" : "display: list-item;")">
-                    @if (IsPictureCard && file.State == UploadState.Uploading)
-                    {
-                        <div class="ant-upload-list-item ant-upload-list-item-uploading ant-upload-list-item-list-type-picture-card">
-                            <div class="ant-upload-list-item-info">
-                                <div class="ant-upload-list-item-thumbnail">@Locale.Uploading</div>
-                                <span class="ant-upload-list-item-name ant-upload-list-item-name-icon-count-1" title="@file.FileName">
-                                    @file.FileName
-                                </span>
-                            </div>
-                            <div class="ant-upload-list-item-progress">
-                                <Progress Percent="file.Percent" StrokeWidth="2" Type="ProgressType.Line" ShowInfo="false"></Progress>
+                    <Unbound>
```

Comment:
I find the `Class` in Unbound usage is not work, and here is missing the `containerClass` so the style is broken.